src/style.css

```diff
@@ -1,3 +1,59 @@
+:root {
+  --main-blue-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px;
```

Comment:
Consider adding a `background-color` to the `header` to ensure it stands out against the rest of the page content.